src/plugins/data_table/server_side_data_table.js

```diff
@@ -88,6 +88,71 @@ var ServerSideDataTable = (function() {
       this.trigger("draw", arguments);
     },
 
+    exportData : function(sUrl) {
+      var oSettings = this.dataTable.fnSettings;
+      var aoData = this.dataTable._fnAjaxParameters( oSettings );
+      this._addServerParams( aoData );
+      this._fetchCSV( sUrl, aoData );
+    },
+
+    _fetchCSV : function (sUrl, aoData) {
+      if (this.serverSideFiltering) {
+        var filterJson = {};
+        filterJson.name = "ext_filter_json";
+        filterJson.value = this._getFilteringSettings();
+        aoData.push(filterJson);
+      }
+      window.location = sUrl;
+      /*$.ajax({
+        url: sUrl,
+        data : aoData,
+        dataType : "text",
+        cache : false,
+        type : this.ajaxMethod || "GET",
+        success: function(response, status, xhr) {
+          // check for a filename
+          var filename = "";
+          var disposition = xhr.getResponseHeader('Content-Disposition');
+          if (disposition && disposition.indexOf('attachment') !== -1) {
+            var filenameRegex = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/;
+            var matches = filenameRegex.exec(disposition);
+            if (matches != null && matches[1]) filename = matches[1].replace(/['"]/g, '');
+          }
+
+          var type = xhr.getResponseHeader('Content-Type');
+          var blob = new Blob([response], { type: type });
+
+          if (typeof window.navigator.msSaveBlob !== 'undefined') {
+            // IE workaround for "HTML7007: One or more blob URLs were revoked by closing
+            // the blob for which they were created. These URLs will no longer resolve as
+            // the data backing the URL has been freed."
+            window.navigator.msSaveBlob(blob, filename);
+          } else {
+            var URL = window.URL || window.webkitURL;
+            var downloadUrl = URL.createObjectURL(blob);
+
+            if (filename) {
+              // use HTML5 a[download] attribute to specify filename
+              var a = document.createElement("a");
+              // safari doesn't support this yet
+              if (typeof a.download === 'undefined') {
+                window.location = downloadUrl;
+              } else {
+                a.href = downloadUrl;
+                a.download = filename;
+                document.body.appendChild(a);
+                a.click();
+              }
+            } else {
+              window.location = downloadUrl;
+            }
+
+            setTimeout(function () { URL.revokeObjectURL(downloadUrl); }, 100); // cleanup
+          }
+        }
+      })*/
```

Comment:
Well the main reason I commented this here is because I want to know if the functionality without it is what we intended, as per my comment yesterday before b109198.  I will definitely delete it if we're good on that.